src/hooks/useDataSource.tsx

```diff
@@ -176,13 +176,11 @@ async function createEmptyBook() {
 }
 
 async function loadStockerPrices() {
-  if (IS_PAID_PLAN) {
-    try {
-      // We have to await because we use Price.upsert. Once we correct this,
-      // saving Price already updates the local cache so we don't need await
-      await insertTodayPrices();
-    } catch (e) {
-      console.warn(`Retrieving live prices failed ${e}`);
-    }
```

Comment:
given we re-use the insertTodayPrices somewhere else, just move the logic of `IS_PAID_PLAN` within.